packages/twenty-server/src/engine/metadata-modules/field-metadata/utils/validate-default-value-for-type.util.ts

```diff
@@ -69,13 +69,24 @@ export const validateDefaultValueForType = (
       FieldMetadataClassValidation
     >(validator, conputedDefaultValue as FieldMetadataClassValidation);
 
-    return (
-      validateSync(defaultValueInstance, {
-        whitelist: true,
-        forbidNonWhitelisted: true,
-        forbidUnknownValues: true,
-      }).length === 0
-    );
+    const errors = validateSync(defaultValueInstance, {
+      whitelist: true,
+      forbidNonWhitelisted: true,
+      forbidUnknownValues: true,
+    });
+
+    const hasErrors = errors.length > 0;
+
+    if (hasErrors) {
+      console.error(
```

Comment:
yes use the logger. I would return { isValid, errors } here and log where this util is used using the logger. It should not be the responsability of this util to log. It should either return the errors or throw them